site/content/docs/main/troubleshooting/common-errors.md

```diff
@@ -0,0 +1,65 @@
+# Troubleshooting common errors
```

Comment:
I want to change this title to `Troubleshooting common __ errors` (since I think it should be focused on user facing errors rather than operational errors), maybe `application`? other ideas: `backend`, `upstream`, `proxy`, just `HTTP` (though this might exclude other TCP/TLS etc. specific errors)

Reply:
`traffic`?